website/docs/r/vapp_container.html.markdown

```diff
@@ -0,0 +1,119 @@
+---
+layout: "vsphere"
+page_title: "VMware vSphere: vsphere_vapp_container"
+sidebar_current: "docs-vsphere-resource-compute-vapp-container"
+description: |-
+  Provides a vSphere vApp container resource. This can be used to create and manage vApp container.
+---
+
+# vsphere\_vapp\_container
+
+The `vsphere_vApp_container` resource can be used to create and manage
+Virtual Appliances.
+
+For more information on vSphere Virtual Appliances, see [this
```

Comment:
Terminology also here. Might be worth it to scan the doc for all reference of "Virtual Appliance" and just replace with "vApp".